demos/security_barrier_camera_demo/grid_mat.hpp

```diff
@@ -22,6 +22,9 @@ class GridMat {
             maxWidth = std::max(maxWidth, static_cast<size_t>(sizes[i].width));
             maxHeight = std::max(maxHeight, static_cast<size_t>(sizes[i].height));
         }
+        if (0 == maxWidth || 0 == maxHeight) {
```

Comment:
I'm not sure this is the correct check. Shouldn't we make sure that _none_ of the input sizes are zero?

Reply:
Having one of inputs with zero resolution can be strange, but it still can be processed by GridMat. While all zero inputs will result in division by zero